lib/cli/src/commands/package/common/macros.rs

```diff
@@ -0,0 +1,79 @@
+macro_rules! make_pb {
```

Comment:
I'd put this macro a bit more descriptive: `make_spinner`.

Reply:
https://github.com/wasmerio/wasmer/pull/4643/commits/64e969053f7124302e86fbb97c431abd86f99fb2#diff-171b04fbf128026d96a51378fbf46e7b2609e4aa69a86858705d394bb988fcb7R1